index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>class Grid</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <section class="parent">
+         <section>
+             <div>Logo</div>
+             <div>side bar Navigation</div>
+             <div>
+                <audio src="../../PC/Music/#الشيخ_#خالد_الراشد__يا_امة_#محمد_كامله(128k).m4a" width="500px" controls loop poster="" >
+                <!-- <source src="../../PC/Videos/instagram_video.mp4"> -->
+                </audio>
+              </div>
+             <div> 
+                   <video src="../../PC/Videos/instagram_video.mp4" width="500px" controls loop poster="./assests/images/travel3.jpeg" >
```

Comment:
Here, can't connect to the file form otehr system @Alimyusstore

Reply:
i have fixed it